ingestion/functions/parsing/ch_zurich/zurich.py

```diff
@@ -31,6 +31,16 @@
 _CONFIRMED_INDEX = 4
 _SOURCE_INDEX = 6
 
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in dd-mm-YYYY format, but the date filtering API
```

Comment:
Little confused by this -- it says Y-m-d below, which I thought was correctly handled.

Perhaps this changed since I last tested?

Reply:
Fixed that comment sorry, what changed since last time is the date filtering happening in python.